src/components/utilities.js

```diff
@@ -0,0 +1,17 @@
+export default function formatNoteReferrerEntityData(data) {
```

Comment:
Let's move this file into the new `utils` folder we're creating for the `urls.js` file as well. And rename it something specific like `formatNoteReferrer.js` rather than just `utilities.js`